packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -205,18 +205,17 @@ export class DocumentService implements api.IDocumentService {
 
         // Attempt to establish connection.
         // Retry with new token on authorization error; otherwise, allow container layer to handle.
-        let connection: api.IDocumentDeltaConnection;
         try {
-            connection = await connect();
+            const connection = await connect();
```

Comment:
Is this included because of merge?

Reply:
Yeah. I'm a little confused why it shows up here. It should not be consequential, hopefully

Reply:
It's gone now...